src/main/scala/is/hail/methods/VEP.scala

```diff
@@ -394,7 +394,13 @@ object VEP {
         }
       }.asOrderedRDD
 
-    vsm.copy(rdd = newRDD,
-      vaSignature = newVASignature)
+    if (csq && newVASignature.isInstanceOf[TStruct])
```

Comment:
In this case I kind of like it better this way since otherwise it has a pattern matching + if..